Cargo.toml

```diff
@@ -26,6 +26,7 @@ maplit = "1.0.2"
 sha2 = "0.8.1"
 serde = "1.0.116"
 serde_json = "1.0"
```

Comment:
do we still need the serde_json dep?

Reply:
yes it's used all over the place, e.g., push/pull, and now in indexes.